tests/AiurVersionControl.Tests/BasicModelTests.cs

```diff
@@ -129,5 +135,34 @@ public async Task RemoteWithMergeWorkSpaceTest()
             Assert.AreEqual(1155, repo.WorkSpace.NumberStore);  // 5, 50, 1000, 100
             Assert.AreEqual(1155, repo2.WorkSpace.NumberStore);// 5, 50, 1000, 100
         }
+
+        [TestMethod]
+        public void JsonSerializeTest()
+        {
+            var repo = new ControlledRepository<NumberWorkSpace>();
+            
+            IModification<NumberWorkSpace> modification = new AddModification(100);
+            Object o = modification;
+            var commit = new Commit<IModification<NumberWorkSpace>> {Item = modification};
+            var things = new PushModel<IModification<NumberWorkSpace>>
+            {
+                Commits = new List<Commit<IModification<NumberWorkSpace>>> { commit },
+                Start = "start"
+            };
+
+            string serialize = JsonSerializer.Serialize(things, new JsonSerializerOptions
```

Comment:
No. You can't let users control our class like this. Since our WebSocket wrapper already using the default json settings.